package.json

```diff
@@ -36,6 +36,7 @@
     "gulp-watch": "^4.3.11",
     "istanbul": "^0.4.5",
     "jsdoc-babel": "^0.2.1",
+    "jsdoc-baseline": "git+https://github.com/sugarcrm/jsdoc-baseline.git",
```

Comment:
> Also, aren't we missing the dependency https://github.com/sugarcrm/gulp-jsdoc3?

No, that was already available. https://github.com/sugarcrm/thorn/pull/156/commits/65026ec3a2da3248746ea190db4d3916169ea4c6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R34

> IMHO this should also be an optional dependency, since we don't need this to develop, just to generate the docs.

https://docs.npmjs.com/files/package.json#devdependencies specifically cites documentation frameworks as the sort of thing that should go in devDependencies. If I'm understanding that page correctly, optional dependencies are actually installed by default - the difference is that if they fail to install for some reason, it doesn't cause npm to exit unsucessfully.